sys/dev/athn/if_athn_usb.c

```diff
@@ -250,6 +250,7 @@ tsleep_nsec(const void *ident, int priority, const char *wmesg,
     uint64_t nsecs)
 {
 	uint64_t to_ticks;
+	int tick_nsec = 1000000000 / hz;
```

Comment:
It'll be better to round this division (up/down) than just truncate if the result has a fraction part.

In Linux this is defined as:
/* TICK_NSEC is the time between ticks in nsec assuming SHIFTED_HZ */
#define TICK_NSEC ((NSEC_PER_SEC + HZ / 2) / HZ

See https://elixir.bootlin.com/linux/latest/source/include/vdso/jiffies.h#L9